ssl/statem/statem.c

```diff
@@ -951,3 +951,18 @@ int ossl_statem_export_allowed(SSL *s)
     return s->s3->previous_server_finished_len != 0
            && s->statem.hand_state != TLS_ST_SW_FINISHED;
 }
+
+/*
+ * This function returns 1 if early TLS exporter is ready to export
+ * keying material, or 0 if otherwise.
+ */
+int ossl_statem_export_early_allowed(SSL *s)
+{
+    /*
+     * The early exporter secret is only present on the server if we
+     * have accepted early_data. It is present on the client as long
+     * as we have sent early_data.
+     */
+    return s->ext.early_data == SSL_EARLY_DATA_ACCEPTED
+           || (!s->server && s->ext.early_data != SSL_EARLY_DATA_NOT_SENT);
```

Comment:
Do we need to indent like this?  This is a question.  Usually it's just one level (i.e., under the second 'r' in return)

Reply:
Fixed except for this style change.  Should I do this?
Does OpenSSL project have formatter just like clang-format?